apps/grav/templates/partials/base.html.twig

```diff
@@ -24,13 +24,17 @@
 
 <body>
 
+{% block navbar %}
+  {% include 'partials/navbar.html.twig' %}
+{% endblock %}
+
 <header class="site__header" role="banner">
   {% block header %}
     {% embed '@molecules/jumbotron/_jumbotron.twig' with { jumbotron_fluid: true } %}
       {% block jumbotron_branding %}
         {% include '@atoms/branding/_branding.twig' with {
-          site_name: 'Particle in ' ~ site.title|e('html'),
-          url: '/',
+          site_name: site.title|e('html'),
```

Comment:
What is the `e()` twig filter here?

Reply:
I pulled this from the similar work done in Grav's antimatter theme. My guess was down to the fact that site.title could be set in headmatter on an admin menu. So since that value can accept user input, it should be escaped.

Reply:
Ah, `e()` is escape!